File: Searching algos/Linear_Search.py
#List in which we want to perform the Linear Search
array = [1, 2, 3, 5, 7, 9, 11, 15, 19, 34, 35]

#The element which need to be searched
search_element = int(input("Enter the number to search: "))

#defining the function
def linearsearch(array, search_element):                
   for item in range(len(array)):          
      if array[item] == search_element:                 #If the given item is equal to the search element
         return item                                    #then, it will return the index
   return -1                                            #if search element not found in the array it will return -1
result = linearsearch(array, search_element)            #function call
if (result != -1):                                      #output for the above given conditions
    print(f"Element is present at index {result}")
else:
    print("Element is not present in the given array")

File: Searching algos/Linear_Search.java
public class Linear_Search {
    
    public static int linear(int array[], int key)
    {
        int size=array.length;
        int index;
        for (index=0; index<size; index++)
        {
            if (array[index]==key)
            {
                return index;
            }
        }
        return -1;
    }

    public static void main(String[] args)
    {
        int array[]={5, 6, 12, 4, 56, 87, 12};
        int key = 56;
        int index=linear(array, key);
        if (index==-1)
        {
            System.out.println("Element not found.");
        }
        else
        {
            System.out.println("Element found at index " + index);
        }
     }
}
